hphp/tools/oss_hot_section_ordering

```diff
@@ -818,7 +818,7 @@
 .text.hash_string_i_unaligned_crc*
 .text._ZN4HPHP9ArrayData10IsValidKeyERKNS_7VariantE*
 .text._ZN4HPHP21tvMatchesRepoAuthTypeENS_10TypedValueENS_12RepoAuthTypeE*
-.text._ZN4HPHP18IsSSEHashSupportedEv*
+.text._ZN4HPHP17IsHwHashSupportedEv*
```

Comment:
I don't think we often write to this file directly.  The file should be generated from profiling after building.  Since the function is hot, it should be picked up next time we update this list.  On the other hand, it is OK to edit here directly.